hack/istio/playbooks/README.adoc

```diff
@@ -0,0 +1,48 @@
+== Requirements
+
+- pip
+- openshift-python (pip install openshift)
+- ansible 2.5+
+
+
+== Installing Bookinfo
+
+[NOTE]
+This playbook support both Istio 0.8 and Istio 1.0 - check your istioctl version before installing.
+
+- All the Bookinfo Demo pods and services are installed in the "bookinfo" namespace.
+
+- In order to uninstall it, simply delete that namespace via something like `kubectl delete namespace bookinfo` or
+`oc delete project bookinfo`.
+
+- In order quick install, you may use the following command for istio 0.8.0 version
+
+```
+ansible-playbook bookinfo.yml -e istio_version=0.8 -e traffic_generator=false -v
+```
+- For 1.0 version, you may use
+
+```
+ansible-playbook bookinfo.yml -e istio_version=1.0 -e traffic_generator=false -v
+```
+
+- If you want to include traffic generator to bookinfo project, see sections below
+
+=== Including Traffic Generator with Ingress Route
+
+- If you want to use ingress route (all traffic is going to hit ingress-gateway), you may use the following command:
+
+```
+ansible-playbook  playbooks/bookinfo.yml -e istio_version=1.0 -e ingress_route=true -e traffic_generator=true -e rate=1 -vv
```

Comment:
in the first two cmdline examples you give above, you just use "bookinfo.yml", in this cmdline example (and the next one below) you use "playbooks/bookinfo.yml". Should be consistent across all four - since this README is in playbooks folder, I would just use "bookinfo.yml" in these cmdline examples.